Content.Shared/Verbs/Verb.cs

```diff
@@ -180,5 +187,80 @@ public int CompareTo(object? obj)
             // Finally, compare icon texture paths. Note that this matters for verbs that don't have any text (e.g., the rotate-verbs)
             return string.Compare(IconTexture, otherVerb.IconTexture, StringComparison.CurrentCulture);
         }
+
+        /// <summary>
+        ///     Collection of all verb types, along with string keys.
+        /// </summary>
+        /// <remarks>
+        ///     Apparently a <see cref="Type"/> is not serializable, so these strings are needed for networking.
+        ///     Though probably this should use reflection or something.
+        /// </remarks>
+        public static Dictionary<Type, string> VerbTypes = new()
+        {
+            { typeof(Verb), "Verb" },
+            { typeof(InteractionVerb), "InteractionVerb" },
+            { typeof(AlternativeVerb), "AlternativeVerb" },
+            { typeof(ActivationVerb), "ActivationVerb" },
+        };
+    }
```

Comment:
This is probably the jankiest part of this. Apparently `Type` isn't serializable, so AFAIK the easiest way of communicating to the server what types of verbs have been requested is to just turn it into a list of strings and then back again. Though the keys of this dictionary are also useful when enumerating over all verb types,

Reply:
cant you just store a list of verbtypes and then use `nameof()`

Reply:
Yeah I should have done something like that. Now using `Type.Name`.